src/components/DeeplinkWrapper/deeplinkRoutes.js

```diff
@@ -0,0 +1,46 @@
+import ROUTES from '../../ROUTES';
+
+/** @type {Array<object>} Routes regex used for desktop deeplinking */
+export default [
+    {
+        // /reports/*
+        pattern: `/${ROUTES.REPORT}($|(//*))`,
+    },
+    {
+        // /settings/*
+        pattern: `/${ROUTES.SETTINGS}($|(//*))`,
+    },
```

Comment:
@cowboycito 

Wondering if can we construct this from the apple-site association file, instead of hard coding here as it may come in handy in the future.

cc: @NikkiWines

Reply:
Oh, that sounds like a great call @Santhosh-Sellavel. Best to reduce duplication wherever possible

Reply:
I just tried importing that file to use it as a reference to construct the deeplink routes. 2 issues came up: it's outside the `src` directory and the file doesn't have an extension (which causes for webpack to not know how to load the file). I honestly don't think it's worth to go on that route. Any thoughts? @Santhosh-Sellavel @NikkiWines

Reply:
I am not sure, I will wait for @NikkiWines

Reply:
Hmm, yeah looking into it more I can't find a simple route to get this to work. In which case I think this is fine as it is now.